core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/StackV4Response.java

```diff
@@ -370,6 +373,14 @@ public void setCredentialCrn(String credentialCrn) {
         this.credentialCrn = credentialCrn;
     }
 
+    public Map<String, Set<String>> getServicesOnHostGroups() {
```

Comment:
This information should already be available in the blueprint. See YarnServiceConfigClient and StackResponseUtils.

Reply:
Changed